react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+class NewsLetterForm extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {value: ''};
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event){
+    this.setState({value: event.target.value});
+  }
+
+  handleSubmit(event){
+    console.log("State ", {Email:this.state.value});
+    event.preventDefault();
+  }
+
+  render(){
+    return (
+    <form onSubmit={this.handleSubmit}>
+      <label for="email">Email</label>
+        <input type="email" onChange={this.handleChange} />
```

Comment:
you need to add the value attribute to the input and assign it the value of the value in state so that react maintains control of what is being displayed on the DOM